proto/pulumi/provider.proto

```diff
@@ -89,8 +93,34 @@ service ResourceProvider {
     rpc GetMappings(GetMappingsRequest) returns (GetMappingsResponse) {}
 }
 
+message ParameterizeRequest {
+    message ParametersArgs {
+        repeated string args = 1;
+    }
+
+    message ParametersValue {
+        // The key (a valid package name) for this sub-schema parameterization.
+        string key = 1;
+        // The package version this was sent from.
+        string version = 2;
```

Comment:
I'm a little fuzzy on what package we are talking about here. Is this the version of the generated SDK?

Reply:
Yes, when talking about parameterised providers we need to talk about the base provider plugin version and the version for the generated sub-package. The provider already know's it's own version, so this is the sub-package version.